maven-release-manager/pom.xml

```diff
@@ -238,9 +238,7 @@
         <executions>
           <execution>
             <goals>
-              <goal>xpp3-reader</goal>
```

Comment:
No model necessary anymore?

Reply:
model is useful, ie java classes
but there is no XML read/write with XPP3

Reply:
Alright!